python/tests/test_path_compatibility_all.py

```diff
@@ -19,7 +19,9 @@
 # We will drop this test file in the future when we remove path compatibility for Apache Sedona < 1.8.0
 
 from sedona.spark import *
+from sedona.spark.maps.SedonaMapUtils import SedonaMapUtils
```

Comment:
The whole point of `test_path_compatibilty_all.py` is to test `from sedona.spark import *`. All other tests should remain the same as `test_path_compatibilty.py`